packages/store/tests/selector.spec.ts

```diff
@@ -817,4 +799,55 @@ describe('Selector', () => {
       }
     });
   });
+
+  describe('Select errors', () => {
+    @State<number[]>({
+      name: 'tasks',
+      defaults: [1, 2, 3, 4]
+    })
+    class NumberListState {
+      @Selector()
+      static reverse(state: number[]): number[] {
+        return state.reverse();
+      }
+    }
+
+    it('should be correct catch errors from observable with select', fakeAsync(() => {
+      TestBed.configureTestingModule({
+        imports: [
+          NgxsModule.forRoot([NumberListState], {
+            developmentMode: true,
+            selectorOptions: { suppressErrors: false }
+          })
+        ]
+      });
+
+      const store = TestBed.get(Store);
+      store.reset({ tasks: [1, 2, 3, 4] });
+
+      let snapshot: number[] = [];
+
+      store.select(NumberListState).subscribe((state: number[]) => {
+        snapshot = state;
+      });
+
+      tick(100);
```

Comment:
There are no micro-tasks here, do you need `tick`?

Reply:
remove